scripts/lib/check/0820_hostname.check

```diff
@@ -22,8 +22,8 @@ function check_0820_hostname {
         local short_hostname_from_full
 
         # import short and full hostname
-        short_hostname=$(hostname)
-        full_hostname=$(hostname -f)
+        short_hostname=$(hostname | tr '[:upper:]' '[:lower:]')
```

Comment:
no need for external command 'tr'

just use the '-l' option for the variable declaration by local,

local -l full_hostname
local -l shorthostname

man page for local:
-l     When the variable is assigned a value, all upper-case characters are converted to lower-case.  The upper-case attribute is disabled.